server/src/main/java/org/jboss/as/server/deployment/DeploymentHandlerUtil.java

```diff
@@ -334,4 +351,21 @@ private static String getFormattedFailureDescription(OperationContext context) {
         }
         return failureDesc;
     }
+
+    public static boolean shouldNotify(OperationContext context, ModelNode operation) throws IllegalArgumentException {
+        if(context.isBooting()) {
+            return false;
+        }
+        if (operation.hasDefined(OWNER.getName())) {
```

Comment:
Is it important to not emit the notification sometimes? If the resource records a notification type it's odd to not emit it sometimes. A deployed/undeployed notification seems like something that might be generally useful, particularly when we get around to supporting remote listeners.

If discriminating based on OWNER is necessary, perhaps make that part of the notification data and then have the listener discriminate?